pkg/util/net/tls.go

```diff
@@ -39,6 +39,15 @@ func GetTLSConfig(certFile, keyFile, caFile string) (*tls.Config, error) {
 		Certificates: certs,
 		ClientCAs:    cas,
 		RootCAs:      cas,
+		CipherSuites: Safe,
 	}
 	return tlsConfig, nil
 }
+
+var Safe = []uint16{
+	tls.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
+	tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
+	tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
+	tls.TLS_RSA_WITH_AES_128_CBC_SHA,
+	tls.TLS_RSA_WITH_AES_256_CBC_SHA,
```

Comment:
Ghostunnel has a set of safe cipher suites if you want to add more:
https://github.com/square/ghostunnel/blob/master/tls.go